website/docs/index.html.markdown

```diff
@@ -32,6 +32,14 @@ resource "opentelekomcloud_compute_instance_v2" "test-server" {
 }
 ```
 
+Or using `clouds.yaml` file:
+```hcl
+provider "opentelekomcloud" {
+  cloud = var.cloud_name
+}
+# ...
+```
```

Comment:
Better to move this to Authentication section below "This provider offers 4 means for authentication."

Reply:
Moved clouds.yaml usage to auth section. Though I'm not sure, if some more information should be added